api/edge_api/identities/export.py

```diff
@@ -0,0 +1,181 @@
+import typing
+import uuid
+from decimal import Decimal
+from typing import Dict, List, Tuple
```

Comment:
Is there a reason for using the typing module instead of just using `dict`, `list`, and `tuple`?

Reply:
Mostly to support things like these

`    feature_id_to_uuid: Dict[int, str] = {}
`

Reply:
This runs just fine unless I'm missing something
```python
def example_function(feature_id_to_uuid: dict[int, str] = {}) -> None:
    print(feature_id_to_uuid)
```

Reply:
Oh! I wasn't aware of that! thanks